app/helpers/homes_helper.rb

```diff
@@ -1,2 +1,7 @@
 module HomesHelper
+	def get_star(hotel)
+		@ratings=hotel.ratings.all
+		@ratings_tot= Rating.where(hotel_id: hotel.id).sum(:rating_no)
+        @star=(@ratings_tot.to_i/(@ratings.count*5))*5
```

Comment:
are you trying to get average rating here ? if yes, then you can use hotel.rating.avg(:rating_no).